examples/feature_alignment_example/server.py

```diff
@@ -0,0 +1,98 @@
+import argparse
+from pathlib import Path
+from typing import Any, Dict, List, Tuple
+
+import flwr as fl
+import pandas as pd
+from flwr.common.parameter import ndarrays_to_parameters
+from flwr.common.typing import Metrics, Parameters
+
+from examples.models.logistic_regression import LogisticRegression
+from examples.simple_metric_aggregation import metric_aggregation, normalize_metrics
+from fl4health.client_managers.poisson_sampling_manager import PoissonSamplingClientManager
+from fl4health.feature_alignment.tab_features_info_encoder import TabularFeaturesInfoEncoder
+from fl4health.server.tabular_feature_alignment_server import TabularFeatureAlignmentServer
+from fl4health.strategies.basic_fedavg import BasicFedAvg
+from fl4health.utils.config import load_config
+
```

Comment:
A comment here might be good saying that this data path is used to create a "source of truth" on the server-side as an example. This is used if the config specifies `source_specified` as true